components/logo/css.ts

```diff
@@ -0,0 +1,21 @@
+const logoBase = {
+  topContainer: ['flex', 'items-center', 'justify-center'],
+  logoContainer: ['w-[64px]', 'h-[64px]'],
+};
+
+export const logoLeft = {
+  topContainer: [...logoBase.topContainer, 'flex-row'],
+  logoContainer: [...logoBase.logoContainer, 'mr-6'],
+};
+export const logoTop = {
+  topContainer: [...logoBase.topContainer, 'flex-col'],
+  logoContainer: [...logoBase.logoContainer, 'mb-4'],
+};
+
+export const iconColors = {
+  white: 'text-white',
+  violet: 'text-violet-500',
+  gradient: 'text-violet-500',
```

Comment:
is this a copy paste issue? :-) 
violet and gradient have the same text-color

Reply:
you can use this gradient text color `gradient-pink-violet-5050` which is defined in `tailwind.config.ts`

you might also need to add the tailwind classes `text-transparent` and `bg-clip-text` to the gradient text: https://daily-dev-tips.com/posts/making-gradient-text-with-tailwind-css/

Reply:
You can ignore my previous two comments. I realised that those text-colors are used to set the color of the panda icon and not the mumble text. The mumble text colors are set within the svg files.